README.md

```diff
@@ -165,12 +165,20 @@ print(response.prompt)
 By using the `count_tokens` method, you can estimate the billing for a given request.
 
 ```python
-from ai21 import AI21Client
+from ai21.tokenizers import get_tokenizer
 
-client = AI21Client()
-client.count_tokens(text="some text")  # returns int
+tokenizer = get_tokenizer(name="jamba-instruct-tokenizer")
```

Comment:
Don't you want to update the examples file as well? Currently it uses the deprecated way